README.md

```diff
@@ -229,9 +233,9 @@ flag `--absentees`.  The arguments can be entered in the following ways:
 
 ```shell
 
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --absentees student1 student2
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --absentees 'student1' 'student2'
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --absentees "student1" "student2"
+pipenv run python3 gatorgrouper_cli.py --file filepath --absentees student1 student2
+pipenv run python3 gatorgrouper_cli.py --file filepath --absentees 'student1' 'student2'
+pipenv run python3 gatorgrouper_cli.py --file filepath --absentees "student1" "student2"
```

Comment:
You should only include one option, since these differences are dependent on the `shell` (bash, zsh, etc) that the user is using, and not your program.

Reply:
This should be fixed